src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/RequiredFilesVerifier.java

```diff
@@ -93,6 +94,14 @@ private void checkSecurityScan(GHRepository repo, HashSet<VerificationMessage> h
         }
     }
 
+    private void checkDependabot(GHRepository repo, HashSet<VerificationMessage> hostingIssues) throws IOException {
+        if (fileNotExistsInRepo(repo, ".github/dependabot.yml") &&
```

Comment:
I think this should also check to see if renovate is there too and be silent if its not there.